common/src/main/java/com/gregtechceu/gtceu/client/model/PipeModel.java

```diff
@@ -174,11 +176,16 @@ public TextureAtlasSprite getParticleTexture() {
         return sideSprite;
     }
 
+    private final Function<Optional<Direction>, List<BakedQuad>> modelBaker = Util.memoize(direction ->
```

Comment:
Im not sure whether it works well on a dedicated server

Reply:
Would this even be called on servers? I assume for calculating the hitbox, right?

Reply:
Looked into this now ^^
It's not called on servers, as `renderItems()` is client-only. Keep in mind that this may not work as well as it does in case we cache the baked quads in general in the future.